docs/logs.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Logs and test results
+---
+
+# Logs and test results
+
+When you already wrote your tests and have them running, you need some way to know if the tests passed or need some attention. 
+Patrol has 2 ways of communicating those results - logs in console and native test report.
+
+## Logging every step
+
+<Warning>
+        This feature is available in versions `3.13.0` and later.
+
+        If you're using this version but you don't see any logs, check if you have your own `PatrolTesterConfig` passed 
+        to `patrolTest()`. If yes, add `printLogs: true` argument to this constructor.
```

Comment:
> don't see any logs

If user don't see any logs, than it will be another problem. This flag steer only showing test steps from patrol_finders actions. E.g. native action steps still will be printed.